dev-infra/caretaker/check/github.ts

```diff
@@ -37,14 +47,32 @@ async function getGithubInfo(git: GitClient, {githubQueries: queries = []}: Care
     graphQlQuery[queryKey] = params(
         {
           type: 'ISSUE',
+          first: 100,
           query: `"${repoFilter} ${query.replace(/"/g, '\\"')}"`,
         },
-        {issueCount: types.number},
+        {
+          issueCount: types.number,
+          edges: [{
+            node: {...onUnion({
+              PullRequest: {
+                title: types.string,
+                url: types.string,
+              },
+              Issue: {
+                title: types.string,
+                url: types.string,
+              }
+            })}
+          }]
+        },
     );
   });
   /** The results of the generated github query. */
   const results = await git.github.graphql.query(graphQlQuery);
   Object.values(results).forEach((result, i) => {
     info(`${queries[i]?.name.padEnd(25)} ${result.issueCount}`);
+    for (const edge of result.edges) {
+      info(`  • ${edge.node.url}`);
+    }
```

Comment:
Rather than grabbing more in the search, would it be better to just provide the link to github?

```suggestion
    if (result.issueCount) {
      const url = encodeURI(`https://github.com/${owner}/${repo}/issues?q=${queries[i]?.query}`;
      info(`  - ${url}`)
    }
```

Reply:
It's would be extra clicks to do that when we're capable of providing direct links to those issues (unless there's some cost to doing this). How about both? 

```
Github Tasks
  Merge Queue               9
  https://github.com/angular/angular/issues?q=is:pr%20is:open%20label:%22action:%20merge%22
    - https://github.com/angular/angular/pull/39139
    - https://github.com/angular/angular/pull/39137
    - https://github.com/angular/angular/pull/39019
    - https://github.com/angular/angular/pull/38750
    - https://github.com/angular/angular/pull/38716
    - https://github.com/angular/angular/pull/38665
    - https://github.com/angular/angular/pull/38649
    - https://github.com/angular/angular/pull/37799
    - https://github.com/angular/angular/pull/32084
  Merge Assistance Queue    0
  Initial Triage Queue      0
```
For regular merge queue, this could actually mean caretaker doesn't even have to open GitHub at all.